### ISSUE TITLE
Missing proper screen reader support

### ISSUE THREAD

**Issue by ksb749** [jellyfin/jellyfin-media-player#585]
The Jellyfin Media Player for Windows is missing support for screen readers like NVDA and Jaws. A screen reader is a piece of software that allows blind users to use computers. In general the Jellyfin Media Player has issues, i just installed Jellyfin Media Player 1.9.1 and got thru the installation, but when i launched it, i was not able to use it.

Im running Windows 10 pro and NVDA and Jaws screen readers. The Jellyfin-web works better, im not sue if your code relies on the Jellyfin-web, if, then im sorry for making this issue here and will go to their project instead.

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
